exercise/js/main.js

```diff
@@ -15,22 +15,22 @@
  */
 
 function question1() {
-	$("replaceMe").css("replaceMe", "replaceMe");
+	$("p").css("color", "blue");
 }
 
-$("#button1").on('click', question1);
+$("#button1").on('click', question1());
 
 /**
  * Question 2:
  * When button 1 is clicked, select all <h2> elements on this page, and change the text to say your name.
  * Uncomment the code below and solve the problem
  */
 
-	// function question2(){
-	// 	$("replaceMe").replaceMe("replaceMe");
-	// }
+function question2() {
+	$("h2").text("AFnan Basudan");
+}
 
-	// $("#button1").on('click', question2);
+$("#button1").on('click', question2());
```

Comment:
function binding should be without brackets, else the function will execute without being called.